src/PKSim.Core/Services/PKAnalysesTask.cs

```diff
@@ -40,6 +41,17 @@ public interface IPKAnalysesTask : OSPSuite.Core.Domain.Services.IPKAnalysesTask
 
       IndividualPKAnalysis CalculateFor(Simulation simulation, DataColumn dataColumn);
       PKValues CalculatePK(DataColumn column, PKCalculationOptions options);
+
+      /// <summary>
+      ///    Resolves options and use the mapper to create a PKAnalysis out of the values and a simulation for a given compound
+      /// </summary>
+      /// <param name="pkValues">values to use</param>
+      /// <param name="simulation">the simulation</param>
+      /// <param name="compound">the compound containing its name and molweight</param>
+      /// <returns></returns>
+      PKAnalysis CreatePKAnalysisFromValues(PKValues pkValues, Simulation simulation, Compound compound);
+
+      IReadOnlyList<PopulationPKAnalysis> AggregatePKAnalysis(Simulation populationDataCollector, IEnumerable<QuantityPKParameter> pkParameters, IEnumerable<StatisticalAggregation> selectedStatistics, string captionPrefix);
```

Comment:
I have no idea what this captionPrefix means. But I did not have time to investigate. Probably the name of the output but hard to tell